### PR TITLE
feat: Deploy ACME to infra cluster

### PR DESCRIPTION
**Pull request by tumido**
Requires: https://github.com/operate-first/apps/pull/786
Resolves: https://github.com/operate-first/support/issues/282

/hold
for dependencies

### CODE REVIEW

File: envs/moc/infra/operate-first/acme-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: acme-operator
+spec:
+  destination:
+    name: zero
```
Comment (HumairAK) [inline envs/moc/infra/operate-first/acme-operator.yaml:8]: don't you want to deploy to `infra` ?

Comment (tumido) [reply envs/moc/infra/operate-first/acme-operator.yaml]: good catch, fixed

**Comment by tumido** [comment]
/cc @4n4nd @HumairAK 

please review

**Review by HumairAK — APPROVED** [review]
/lgtm
/approve

**Pull request merged**